domino/routes.py

```diff
@@ -4,20 +4,30 @@ def __init__(self, host, owner_username, project_name):
         self._owner_username = owner_username
         self._project_name = project_name
 
-    # Project URLs
+    # URL builders
     def _build_project_url(self):
         return self.host + '/v1/projects/' + \
             self._owner_username + '/' + self._project_name
 
     def _build_project_url_private_api(self):
-        return self.host + '/u/' + self._owner_username + '/' + self._project_name
+        return self.host + '/u/' + self._owner_username + \
+            '/' + self._project_name
+
+    def _build_old_project_url(self):
```

Comment:
It doesn't seem like we still need this. Should we just yank this out?

Reply:
Talked IRL.. used in some places until D3.0 APIs